src/js/structs/__tests__/ServiceTree-test.js

```diff
@@ -397,7 +545,8 @@ describe('ServiceTree', function () {
         tasksStaged: 1,
         tasksRunning: 4,
         tasksHealthy: 16,
-        tasksUnhealthy: 1
+        tasksUnhealthy: 1,
+        tasksUnknown: -13
```

Comment:
It is weird, but this example will actually never happen, tasksHealthy is always smaller or equal to tasksRunning. I'll change the example to be more representing the real world.